provider/sidecar.go

```diff
@@ -10,21 +10,34 @@ import (
 	"strings"
 	"time"
 
-	fsnotify "gopkg.in/fsnotify.v1"
-
 	"github.com/BurntSushi/toml"
 	"github.com/Nitro/sidecar/catalog"
 	"github.com/Nitro/sidecar/service"
 	"github.com/containous/flaeg"
 	"github.com/containous/traefik/log"
 	"github.com/containous/traefik/safe"
 	"github.com/containous/traefik/types"
+	fsnotify "gopkg.in/fsnotify.v1"
 )
 
 const (
-	method = "wrr"
-	weight = 0
-	sticky = false
+	method               = "wrr"
+	weight               = 0
+	sticky               = false
+	maxConnExtractorFunc = "request.host"
```

Comment:
If you guys really think we need this level of control with the max connections per backend (as opposed to one global setting), then I agree with Mihai's implementation.  Simplest would be to expose it in the toml config file.